### PR TITLE
generate client for clusterprofile api

### PR DESCRIPTION
**Pull request by haoqing0110**
Generate clientset, informers and lister for `ClusterProfile` API.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,25 +5,54 @@ go 1.22.0
 toolchain go1.22.2
 
 require (
-	k8s.io/apimachinery v0.30.0
+	k8s.io/apimachinery v0.30.3
+	k8s.io/client-go v0.29.2
```
Comment (mikeshng) [inline go.mod:9]: Does it have to be this version of the client?

Comment (RainbowMango) [reply go.mod:9]: +1
Yes, it should be.

Comment (RainbowMango) [reply go.mod:9]: We can all start with v0.30.3

Comment (haoqing0110) [reply go.mod]: fixed.

**Comment by mikeshng** [comment]
/approve

THANK YOU Qing Hao! 

Leaving the gtm for another maintainer to take a look.

**Review by mikeshng — APPROVED** [review]
/lgtm

**Pull request merged**